Tests/Unit/Model/EventTest.php

```diff
@@ -18,9 +20,32 @@ class EventTest extends UnitTestCase
      */
     private $subject = null;
 
+    /** @var \Tx_Seminars_Model_Organizer */
```

Comment:
For consistency, generally please use the 3-line PHPDoc formatting for field documentation. (The 1-line form is for local variables only.)